### DISCUSSION TITLE
VAF calculation

### DISCUSSION THREAD

**Discussion by RoniHaas** [uclahs-cds/pipeline-call-sSV#99] [Q&A] [answered]
Hi @Faizal-Eeman!

I want to calculate the VAF for the SV I received. I tried to do that by extracting the relevant VCF fields, but I am a bit confused about the different options offered:

`##FORMAT=<ID=RC,Number=1,Type=Integer,Description="Raw high-quality read counts or base counts for the SV">
##FORMAT=<ID=RCL,Number=1,Type=Integer,Description="Raw high-quality read counts or base counts for the left control region">
##FORMAT=<ID=RCR,Number=1,Type=Integer,Description="Raw high-quality read counts or base counts for the right control region">
##FORMAT=<ID=RDCN,Number=1,Type=Integer,Description="Read-depth based copy-number estimate for autosomal sites">
##FORMAT=<ID=DR,Number=1,Type=Integer,Description="# high-quality reference pairs">
##FORMAT=<ID=DV,Number=1,Type=Integer,Description="# high-quality variant pairs">
##FORMAT=<ID=RR,Number=1,Type=Integer,Description="# high-quality reference junction reads">
##FORMAT=<ID=RV,Number=1,Type=Integer,Description="# high-quality variant junction reads">` 

I thought I should rely on high-quality read counts.
i.e.
RC / (RCL + RCR)

But should the junction reads be taken into account somehow? Are these junction reads included in the high-quality read counts?

Thank you!

**Comment by Faizal-Eeman** [comment, marked as answer]
Hi @RoniHaas,

> But should the junction reads be taken into account somehow? 

Junction reads are not required to calculate VAF as they can be used to confirm the variant call. You can rely on high-quality read counts for VAF using `RC / (RCL + RCR)`.

> Are these junction reads included in the high-quality read counts?

No, they are not included in the high-quality read counts. Junction reads are captured separately to validate the variant calls. If the junction reads appear to support a variant call, then this gives confidence to the variant call.

Best,
Faizal
